PubSub/Tests/LinuxMain.swift

```diff
@@ -0,0 +1,7 @@
+import XCTest
```

Comment:
We can remove the PubSub folders since it's not implemented and we can move these files to a common location.

Reply:
Other than that it looks good to me.

Reply:
Sorry, i forgot those 🙇 will change in the mean while